concrete/src/Entity/Notification/UserDeactivatedNotification.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Concrete\Core\Entity\Notification;
+
+use Concrete\Core\Notification\View\UserDeactivatedListView;
+use Concrete\Core\User\Event\DeactivateUser;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(
+ *     name="UserDeactivatedNotifications"
+ * )
+ */
+class UserDeactivatedNotification extends Notification
```

Comment:
Shouldn't we have a migration? Otherwise the `UserDeactivatedNotifications` database table won't be created.

Reply:
Yeah we have `concrete/src/Updater/Migrations/Migrations/Version20181029223809.php`